cardano-testnet/src/Testnet/Process/Cli/Transaction.hs

```diff
@@ -39,6 +42,11 @@ data TxBody
 
 data SignedTx
 
+data ReferenceScriptJSON
+
+data TxOutAddress = PKAddress PaymentKeyInfo
+                  | ReferenceScriptAddress (File ReferenceScriptJSON In)
```

Comment:
A comment clarifying that the output will be created at the reference script address would be useful. A reference script can exist at any address and it it also an address itself.

Reply:
Ah, I didn't know that